vendor/github.com/gophercloud/gophercloud/openstack/compute/v2/images/doc.go

```diff
@@ -1,7 +1,32 @@
-// Package images provides information and interaction with the image API
-// resource in the OpenStack Compute service.
-//
-// An image is a collection of files used to create or rebuild a server.
-// Operators provide a number of pre-built OS images by default. You may also
-// create custom images from cloud servers you have launched.
+/*
```

Comment:
I don't understand why you introduce all modules of gophercloud? The images and other modules are useless for kubernetes, can we not introduce them?

Reply:
@FengyunPan - I had to add it because verify fails otherwise.

Reply:
All right.